travis/test_server.py

```diff
@@ -463,6 +463,25 @@ def main(argv=None):
         return 1
     elif counted_errors != expected_errors:
         return 1
+    # no test error, let's generate .pot and msgmerge all .po files
+    must_run_makepot = (
+        os.environ.get('MAKEPOT') == '1' and
+        os.environ.get('TRAVIS_REPO_SLUG', '').startswith('OCA/') and
+        os.environ.get('TRAVIS_BRANCH') in ('8.0', '9.0', '10.0', '11.0') and
```

Comment:
What happens with 7.0 and 6.1 branches? I understand that only PRs for translation files, isn't it?

Reply:
Weblate would work without changes. We need to make sure to have clean .pot files though. And that is manual operation for now for 6.1 and 7.0. I think it's not worth the effort to automated it at this stage.

Reply:
Yes, I agree. It was just for being sure.